tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj

```diff
@@ -18,6 +18,14 @@
     <PackageReference Include="System.Diagnostics.Process" Version="4.3.0" />
   </ItemGroup>
 
```

Comment:
as discussed in our PR reviews, this would be a good place for a comment explaining that we are mapping FW versions to Rabbit versions.

Reply:
Good idea.  Done.